alias.txt

```diff
@@ -146,18 +168,23 @@
 .baser Join right base runway $uc($1).
 .fin Join final runway $uc($1).
 .wake Caution wake turbulence.
+.cof Approved to leave the frequency.

```

Comment:
What does `cof` stand for? This is IMHO not very intuitive. Either `atl` or `alf` would make more sense.

Reply:
Im Geiste des Präfix-Suffix Systems wäre das `.co` als Präfix und `f` als Suffix für "frequency". Wobei - wenn ich so darüber nachdenke - `.col` sinnvoller wäre mit `l` für "leave". 🤔

Reply:
Ich habe es jetzt mal in `.col` geändert.